Pully Penelope/Assets/Scripts/Death.cs

```diff
@@ -15,10 +15,18 @@ public class Death : MonoBehaviour
     public bool isDead = false;
     private bool isAlreadyDying = false;
 
+    [Tooltip("The sound to be made upon destruction.")]
+    [SerializeField]
+    private AudioSource deathSound;
+
     private void Start()
     {
         isDead = false;
         spriteRenderer = GetComponent<SpriteRenderer>();
+        if (deathSound == null)
+        {
```

Comment:
Instead of finding the deathsound in the scene could you serialize it in the editor as an audioclip? This would save you from calling a .Find and and GetCompenent, which both tend to be expensive calls

Reply:
So the reason why I had to do this is because it is attached to the player, and the player gets instantiated insead of being in the scene by default. So that's why I couldn't attach it in the inspector, although it would be nice to have a better way.